lib/compiler/src/v3_core.erl

```diff
@@ -123,6 +126,9 @@
 -record(igen,      {anno=#a{},acc_pat,acc_guard,
                     skip_pat,tail,tail_pat,arg,
                     refill={nomatch,ignore}}).
+-record(izip,      {anno=#a{},len,acc_pats,acc_guard,
```

Comment:
Nitpick: we only appear to use `len` once, and set it by taking the length of the list of generators used to make this `#izip{}`. Perhaps we can just take the length of `acc_pats` instead, when required?